src/editorKit.ts

```diff
@@ -339,7 +339,8 @@ export default class EditorKitBase {
 
         if (result.html) {
           note.content.preview_html = result.html
-        } else if (result.plain) {
+        }
+        if (result.plain) {
```

Comment:
Although what if the user is deleting the note's contents and result.plain is an empty string—wouldn't this check fail and the preview would have outdated text?

Reply:
Yeah, I think I've seen this behavior before.